config/crd/bases/devops.kubesphere.io_pipelineruns.yaml

```diff
@@ -36,97 +36,20 @@ spec:
         spec:
           description: PipelineRunSpec defines the desired state of PipelineRun
           properties:
-            parameters:
-              description: Parameters are some key/value pairs passed to runner.
-              items:
-                description: Parameter is an option that can be passed with the endpoint
-                  to influence the Pipeline Run
-                properties:
-                  name:
-                    description: Name indicates that name of the parameter.
-                    type: string
-                  value:
-                    description: Value indicates that value of the parameter.
-                    type: string
-                required:
-                - name
-                - value
-                type: object
-              type: array
-            scm:
-              description: Scm is a SCM configuration that target pipeline run requires.
-              properties:
-                refName:
-                  description: RefName indicates that SCM reference name, such as
-                    master, dev, release-v1.
-                  type: string
-                refType:
-                  description: RefType indicates that SCM reference type, such as
-                    branch, tag, pr, mr.
-                  type: string
-              required:
-              - refName
-              - refType
-              type: object
+            name:
```

Comment:
@LinuxSuRen , please see changes of this file.

Reply:
Older was generated from Jenkins PipelineRun, however, Newer was generated from Tekton PipelineRun.

Reply:
I guess the Kubernetes community might have a solution to fix this problem. Considering @JohnNiang and @ flamywhale are facing this issue. I'd suggest that you try to find a solution for it.

Reply:
According to [this issue](https://github.com/kubernetes/apiextensions-apiserver/issues/17#issuecomment-338005878) in kubernetes community and the [CRD versions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/) official document, I found that there could be multiple versions of one CRD, but one and only one version could be marked as `storage` version hold on the cluster. There is no way to make multiple versions of a CRD to work togather.

Reply:
That's ok. But how to fix the YAML config conflicts issue. Do you have any ideas?